presto-hive-metastore/src/main/java/com/facebook/presto/hive/metastore/thrift/ThriftMetastoreUtil.java

```diff
@@ -135,6 +135,8 @@
     private static final String PUBLIC_ROLE_NAME = "public";
     private static final String ADMIN_ROLE_NAME = "admin";
 
```

Comment:
comment not addressed: https://github.com/prestodb/presto/pull/17830/files#r898715188